tests/integration/targets/aws_glue_job/defaults/main.yml

```
@@ -0,0 +1,2 @@
+glue_role_name: '{{ resource_prefix }}-role'
```

```suggestion
glue_role_name: 'ansible-test-{{ resource_prefix }}'
```

A quirk of our CI permissions, this needs to start with ansible-test-